packages/itwin/imodel-react-hooks/src/FeatureOverrides/useFeatureOverrides.tsx

```diff
@@ -4,13 +4,8 @@
 *--------------------------------------------------------------------------------------------*/
 
 
-import { Logger } from "@bentley/bentleyjs-core";
-import {
-  FeatureOverrideProvider,
-  FeatureSymbology,
-  IModelApp,
-  Viewport,
-} from "@bentley/imodeljs-frontend";
+import { Logger } from "@itwin/core-bentley";
```

Comment:
@MichaelBelousov I tested the useMarker hook, could you test useFeatureOverride ?

Reply:
yea bad practice, but neither of us have the time atm to spend too much time on it. im thinking since we're releasing this pkg as 0.x, if it breaks its not the end of the world

Reply:
i was trying to resolve some test failures, im just going to skip the ones that fail for now. will need anthony to report on the feature override hook later